alex_questions.md

```diff
@@ -0,0 +1,9 @@
+# direct TX
+
+- What format can we expect for the direct TX on the DA? some wrapper type would be useful to unpack and distinct from
+  random bytes
+- Should we always include direct TX although they may be duplicates to mempool TX? spike: yes
+- Should we fill the block space with directTX if possible or reserve space for mempool TX
+
```

Comment:
What should we do when a sequencer was not able to add a direct-TX within the time window?